content/02.main-text.md

```diff
@@ -129,8 +129,7 @@ The notebook to generate this figure can be [interactively launched using Binder
 GitHub issues can also be used for formal peer review by independent or journal-selected reviewers.
 A reviewer conducting open peer review can create issues using their own GitHub account, as [one reviewer](https://github.com/greenelab/meta-review/issues/124) did for this manuscript.
 Alternatively, a reviewer can post feedback with a pseudonymous GitHub account or have a trusted third party such as a journal editor post their comments anonymously.
-Authors can elect to respond to reviews in the GitHub issues or a public [response letter](https://github.com/greenelab/meta-review/blob/813f461fe719d8fe7bcda78871cfad106f83dfe7/content/response-to-reviewers.md), creating open peer review.
-** TODO: update `response-to-reviewers.md` URL once it is finalized **
+Authors can elect to respond to reviews in the GitHub issues or a public [response letter](https://github.com/greenelab/meta-review/blob/v3.0/content/response-to-reviewers.md), creating open peer review.
```

Comment:
The tag works well here.  In the `Continuous publication` section we also have a link to a versioned URL.  Should update that to show a draft that includes all the authors?

Reply:
Currently, the link is to https://greenelab.github.io/meta-review/v/4b6396bcefd1b9c7ddf39c1d3f0b3eab2dd63f31/. I was actually thinking of going even older, although think the current one is fine. Basically, I think it's more meaningful to show a version that is different. Having fewer authors is kind of cool because it demonstrates the more continuous nature of Manubot manuscripts.

Reply:
Good point.  As long as it is a deliberately selected version, I'm okay with it.  I do like the idea of showing major changes to the manuscript.